src/app/login/login.component.html

```
@@ -1,27 +1,32 @@
 <div class="login">
   <form class="login__form" [formGroup]="loginForm">
+    <h2 class="login__form-title login__form-title_center login__form-title_thin">Login</h2>
     <label class="login__form-name">
-      <span class="login__form-title">name</span>
-      <span class="login__form-tooltip">* Can't change after login!</span>
+      <span class="login__form-title">username</span>
       <input type="text" class="login__form-input" formControlName="name"
-        required>
+        required (focus)="hideErrors()">
     </label>
     <label class="login__form-password">
       <span class="login__form-title">password</span>
       <input type="password" class="login__form-input" formControlName="password"
-        required>
+        required (focus)="hideErrors()">
```

Why do we need focus function here?

---

And why do we need to use the `required` attribute? 
Because as I remember correctly for reactive for, you can omit it.

---

https://angular.io/guide/reactive-forms#step-2-making-a-field-required
I think, I can use as `required` as `Validators.required`

---

Focus function needs for hide error messages.